source/views/tasks_view.py

```diff
@@ -164,3 +166,14 @@ def get_statuses(self):
         statuses_data = self.status_schema.dump(statuses, many=True).data
         return jsonify({'statuses':statuses_data})
 
+
+    def get_users_per_task(self, id_task):
+        authorization(request.headers.get('Authorization', None))
+        collaborators = TaskOwner.query.filter_by(id_task=id_task)\
```

Comment:
I think we should use `filter` most of the time. It's more powerfull and more "pythonizable". The other one it's more SQL oriented.

https://stackoverflow.com/questions/2128505/whats-the-difference-between-filter-and-filter-by-in-sqlalchemy